src/Pie.js

```diff
@@ -5,6 +5,15 @@ import {accessor, assign, configPrep, elem} from "d3plus-common";
 import {Path} from "d3plus-shape";
 import {Viz} from "d3plus-viz";
 
+/**
+    @function constructAriaLabel
+    @desc Returns value for aria-label property of Shapes.
+    @private
+*/
+function constructAriaLabel(d, i) {
+  return d.index + ". " + this._drawLabel(d, i) + ", " + d.value + ".";
```

Comment:
@rbaheti, please take a moment to configure your text editor with an [eslint plugin](https://github.com/Microsoft/vscode-eslint). the preferred method for string concatenation as defined by our styleguide (the `.eslintrc` file) is to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), so this code shows a warning. It should be this:

```js
return `${d.index}. ${this._drawLabel(d, i)}, ${d.value}.`;
```